### PR TITLE
Atomic PR 1 or 2 - Toilet Loot!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

That gray tide and clown are at it again and stashing loot in the most odd of places again... The toilet, this pr simply makes the loot tables and toilet subtypes ready to be used by mappers.

## Why It's Good For The Game

This allows some more Rng and loot in the game as well as helps add to the story of the rounds left over. Kinda like the Black Box, Hidden bags, and Photo book. Without all the nasty powergaming and player added op magic bullshit loot that got Black Box and hidden bags storage from round to round removed.

## Changelog
:cl:
add: Map wise you now can have loot tables of, Cigs, Coins, Cigar Cases, Space Cash, Illegal Pills.
add: Mappers can now use toilets that have loot inside them to add a bit more Rng to their station/ruin ect!
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -386,3 +383,141 @@
...
+		/obj/effect/spawner/lootdrop/coin = 5,
+		/obj/effect/spawner/lootdrop/cig_packs = 10,
+		/obj/effect/spawner/lootdrop/cigars_cases = 5,
+		/obj/item/reagent_containers/food/snacks/grown/cannabis = 5,
+		/obj/item/storage/pill_bottle/dice = 5,
+		/obj/item/toy/cards/deck = 5,
+		/obj/effect/spawner/lootdrop/druggie_pill = 5,
+		/obj/item/kitchen/knife = 5,
+		/obj/item/screwdriver = 5,
+		/obj/item/crowbar/red = 0.5, //Dont your need a crowbar to open this?
```
Comment (silicons) [inline code/game/objects/effects/spawners/lootdrop.dm]: spellcheck

File: code/game/objects/structures/watercloset.dm [hunk 2]
```diff
@@ -120,8 +116,14 @@
 		w_items += secret.w_class
 		contents += secret
 
+/obj/structure/toilet/secret/prison //We care about you Skyrats!
```
Comment (silicons) [inline code/game/objects/structures/watercloset.dm]: remove the comment

Comment (ghost) [reply code/game/objects/structures/watercloset.dm]: > 
> 
> remove the comment

@kevinz000 What exactly is wrong with adding a comment that nods to one of your downstreams?

Comment (Putnam3145) [reply code/game/objects/structures/watercloset.dm]: comments are for clarifying what code is doing or why it's doing it and this does neither, it just sorta says we care about skyrat, which, yeah, we totally should, but without the context of this PR that comment makes zero sense

**Comment by Ludox235** [comment]
wtf if this pr gets merged this means i can't hide the spare ID in toilets whithout the risk of a clown trying to get pills finding it

**Comment by KathrinBailey** [comment]
Since you're touching loot spawners, if you really really want to this would be helpful:
10% chance falsewall, 90% wall
25% chance falsewall, 75% wall
50% chance falsewall, 50% wall
75% chance falsewall, 25% wall
I'm not sure what happens once you spawn a closed turf on an open turf but I assume it just replaces the open turf. It'd need to be quickly tested. If you could do that too that'd be 100% amazing and awesome, but you don't have to do it.
Also maybe some stuff that's 10% chance a basic tool 90% chance nothing would be cool too, but I guess that's a lot of stuff outside the scope of your PR. I'd love you forever if you did it though.

**Review by silicons — APPROVED** [review]

**Pull request merged**